packages/cloud_firestore/cloud_firestore_platform_interface/pubspec.yaml

```diff
@@ -8,13 +8,15 @@ environment:
   flutter: ">=1.9.1+hotfix.5 <2.0.0"
 
 dependencies:
-  collection: ^1.15.0
   firebase_core: ">=0.8.0-1.0.nullsafety.0 <0.8.0-2.0.nullsafety.0"
   flutter:
     sdk: flutter
   meta: ^1.3.0-nullsafety.6
   plugin_platform_interface: ^1.1.0-nullsafety.1
 
+dependency_overrides:
```

Comment:
This needs reverting also as discussed on slack collection should stay above in deps as:

```yaml
  collection: ^1.15.0-nullsafety.5
```

Reply:
@Salakar  Am I wrong about this? I thought I thought `1.15.0` was the latest null safe version: https://github.com/dart-lang/collection/commit/9967dcd3d7645db6de48d5abfab3018bb0c84236 🤔 

Will update 👍

Reply:
Nope not wrong, it's just other dependencies in FF are still using `^1.15.0-nullsafety.5` which was causing that bootstrap failure. The reason we don't use `dependency_overrides` is because you can't publish a package that has them in; you can see what happens by doing:

```
cd packages/firebase_auth/firebase_auth
dart pub publish --dry-run
```

`firebase_auth` has one as there was no other alternative to working arounnd the `http_parser` issue, so before publishing we have to remove the overrides every time and then add them back in after publishing. Would rather not have more of these if there's a way to have it working without, e.g. using the previous NNBD version.

Reply:
Thanks for the in-depth explanation 👌